include/xtensor-io/xblosc.hpp

```diff
@@ -0,0 +1,193 @@
+/***************************************************************************
+* Copyright (c) Wolf Vollprecht, Sylvain Corlay and Johan Mabille          *
+* Copyright (c) QuantStack                                                 *
+*                                                                          *
+* Distributed under the terms of the BSD 3-Clause License.                 *
+*                                                                          *
+* The full license is in the file LICENSE, distributed with this software. *
+****************************************************************************/
+
+#ifndef XTENSOR_IO_BLOSC_HPP
+#define XTENSOR_IO_BLOSC_HPP
+
+#include <fstream>
+
+#include "xtensor/xadapt.hpp"
+#include "blosc.h"
+
+bool blosc_initialized = false;
+
+namespace xt
+{
+    namespace detail
+    {
+        char* load_blosc_file(std::istream& stream, std::size_t& uncompressed_size)
```

Comment:
Should be inline to avoid duplicated symbols leading to link errors.

Reply:
:+1: